introduction/index.rst

```diff
@@ -4,192 +4,155 @@ Introduction to the SOF Project
-   driver and drivers for any codec(s) together so they appear as a single
-   ALSA sound card. |SOF| can reuse existing upstream machine drivers (as
-   only the platform name needs to be changed) or can have bespoke machine
-   drivers.
 
-#. **Generic PCM Driver.** The PCM driver creates ALSA PCMs, DAPM, and
-   kcontrols based on the topology data loaded at run time. The PCM driver
-   also allocates buffers for DMA and registers with run time PM. It is
-   architecture and platform generic code.
+Host OS Drivers
+---------------
 
-#. **Generic IPC driver.** The IPC driver is the messaging bridge
-   between the host and DSP and defines the messaging ABI and protocol. It
-   is architecture and platform generic code.
+SOF can be configured and controlled by a host OS driver or can optionally run
+as a stand alone firmware. SOF host drivers currently supports Linux OS today. 
 
-#. **DSP Platform Driver.** The platform driver is a platform specific
-   driver that abstracts the low level platform DSP hardware into a common
-   generic API that is used by the upper layers. This includes code that
-   will initialize the DSP and boot the firmware.
+The SOF driver has a modular stack based architecture that is dual licensed
+BSD & GPL code allowing it to be ported to other OSes and RTOSes. 
 
+The host driver is responsible for :-
 
-   ..	figure::  images/driver-arch-diag.png
-	:align: center
-	:alt: SOF Driver Architecture
-	:width: 800px
+#. Loading firmware from host file system into DSP memories and booting.
 
-	`Sound Open Firmware Driver Architecture`
+#. Loading topologies from host file system into DSP.
 
-The right-hand side of the diagram shows the mailbox/doorbell mechanism and the DSP.
+#. Exposing audio control devices to applications.
 
-The PCM and IPC drivers can be reused without modification on every
-platform. The platform differentiation will occur via the topology data
-and firmware. There is also scope for differentiation via the machine
-driver and platform driver. The ACPI or Device Tree could be used to
-specify the HW configuration.
+#. Exposing audio data endpoints to applications.
 
-FAQ
-===
+#. Managing IPC communication between host and DSP.
 
-What license does the firmware and SDK use?
-  The firmware is released using a standard BSD license with some parts
-  MIT. The SDK is GPL.
+#. Abstraction of host side DSP hardware to common API operations.
 
-Do I need to open source my firmware code changes?
-  No. The firmware BSD and MIT licensed code means you can keep code
-  changes private. Patches are always welcomed if do decide to open source
-  work.
+The Linux SOF ALSA/ASoC driver is upstream in Linux v5.2 onwards.
+
+
+Firmware Toolchain
+------------------
+
+GNU GCC can be used as a free SOF compiler alongside proprietary DSP vendor
+compilers. The choice of compiler is up to the user depending on features
+and budget. GCC complier is free.
```

Comment:
free is repeated here, not necessary. Also it's open-source, not free we should be using.